doc/research.rst

```diff
@@ -0,0 +1,11 @@
+Glossary
+========
+
+`PAC: Practical Accountability for CCF <https://arxiv.org/abs/2105.13116>`_
+  Permissioned ledger systems execute transactions on a set of replicas governed by members of a consortium. They use Byzantine fault tolerance protocols to distribute trust among the replicas, and thus can ensure linearizability if fewer than 1/3 of the replicas misbehave. With more misbehaving replicas, current systems provide no guarantees, and all replicas and members share the blame.
```

Comment:
Should we have a "such as CCF" here?

Reply:
Also "Byzantine-fault tolerant" or perhaps "Byzantine-fault-tolerant" or something but not "tolerance" I would think?

Reply:
This is verbatim from the paper's abstract so I would prefer to keep the two in sync, but we can change the paper text and when we iterate it we will update the text here.